flink-connector-oracle-cdc/src/test/java/com/ververica/cdc/connectors/oracle/table/OracleConnectorITCase.java

```diff
@@ -58,24 +52,24 @@
 import java.util.stream.IntStream;
 import java.util.stream.Stream;
 
+import static com.ververica.cdc.connectors.oracle.source.OracleSourceTestBase.CONNECTOR_PWD;
+import static com.ververica.cdc.connectors.oracle.source.OracleSourceTestBase.CONNECTOR_USER;
+import static com.ververica.cdc.connectors.oracle.source.OracleSourceTestBase.ORACLECONTAINER;
 import static com.ververica.cdc.connectors.oracle.source.OracleSourceTestBase.assertEqualsInAnyOrder;
-import static com.ververica.cdc.connectors.oracle.utils.OracleTestUtils.CONNECTOR_PWD;
-import static com.ververica.cdc.connectors.oracle.utils.OracleTestUtils.CONNECTOR_USER;
+import static com.ververica.cdc.connectors.oracle.source.OracleSourceTestBase.createAndInitialize;
+import static com.ververica.cdc.connectors.oracle.source.OracleSourceTestBase.getJdbcConnection;
 import static org.hamcrest.Matchers.containsInAnyOrder;
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertThat;
 
 /** Integration tests for Oracle redo log SQL source. */
 @RunWith(Parameterized.class)
-public class OracleConnectorITCase extends AbstractTestBase {
+public class OracleConnectorITCase {
```

Comment:
Why not extend `OracleSourceTestBase`?

Reply:
There is no need to extend,because of `OracleSourceTestBase`'s beforeclass is useless for this test.